misc/tools/install.sh

```diff
@@ -152,21 +159,20 @@ echo
 echo "checking composer..."
 
 COMPOSER="$vz_dir/composer.phar"
-if [ ! -f "$COMPOSER" ]; then
+if [ ! -e "$COMPOSER" ]; then
 	for f in composer composer.phar; do
 		COMPOSER=$(which $f 2>/dev/null || true)
-		test -f "$COMPOSER" && break
+		test -n "$COMPOSER" && break
 	done
 fi
-if [ -f "$COMPOSER" ]; then
-	echo "composer: $COMPOSER"
-else
+if [ ! -e "$COMPOSER" ]; then
 	pushd "$vz_dir"
 		echo "composer not found, downloading..."
 		php -r "eval('?>'.file_get_contents('https://getcomposer.org/installer'));"
 		COMPOSER="$vz_dir/composer.phar"
```

Comment:
Not sure here- do we need a `chmod +x` after the get as we're calling composer directly?